main.py

```diff
@@ -74,6 +77,7 @@ def __call__ (self, parser, namespace, values, option_string = None):
     # try this if CUDA OOM
     parser.add_argument('--fp16', action='store_true', help="use float16 for training")
     parser.add_argument('--vram_O', action='store_true', help="optimization for low VRAM usage")
+    parser.add_argument('--batch_size_1', action='store_true', help="render only one image using NeRF")
```

Comment:
I think it would be cleaner to use an `int` `num_images_per_batch` that defaults to 1. Then people with lots of VRAM could crank it up to increase performance and/or quality.